Marlin/language_en.h

```diff
@@ -561,22 +561,35 @@
     #define MSG_INFO_PRINT_COUNT              "Print Count"
   #endif
   #ifndef MSG_INFO_COMPLETED_PRINTS
-    #define MSG_INFO_COMPLETED_PRINTS         "Completed  "
+    #define MSG_INFO_COMPLETED_PRINTS         "Completed"
   #endif
   #ifndef MSG_INFO_PRINT_TIME
-    #define MSG_INFO_PRINT_TIME               "Total Time "
+    #define MSG_INFO_PRINT_TIME               "Total print time"
+  #endif
+  #ifndef MSG_INFO_PRINT_LONGEST
+    #define MSG_INFO_PRINT_LONGEST            "Longest job time"
+  #endif
+  #ifndef MSG_INFO_PRINT_FILAMENT
+    #define MSG_INFO_PRINT_FILAMENT           "Extruded total"
```

Comment:
Well, current (newest RCBugFix) displaying of "Total time" is as follows for some reason...
By #4374? or the others?
![dscn1715](https://cloud.githubusercontent.com/assets/15308844/17075609/bdf33260-50d3-11e6-8ade-fb05b8c190f7.JPG)